docs/software/programming.md

```diff
@@ -132,6 +133,15 @@ Official documentation: <https://documentation.portainer.io>
 Source code: <https://github.com/portainer/portainer>  
 Open-source license: [zlib](https://github.com/portainer/portainer/blob/develop/LICENSE)
 
+## VSCodium
+
+VSCodium is a FLOSS version of [Microsoft's Visual Studio code](https://code.visualstudio.com/), built directly from the source on GitHub, without branding, tracking, or telemetry.
+
+![VSCodium screenshot](../assets/images/dietpi-software-programming-vscodium.png){: width="1028" height="799" loading="lazy"}
+
+Documentation (Visual Studio Code): <https://code.visualstudio.com/docs>
```

Comment:
Please add two spaces at the end of the line to have a correct newline between line 142 and 143.